### PR TITLE
Convert CONTRIBUTORS into an .md file

### LINKED ISSUE #193: Convert contributors file to markdown file
so it looks better… :smile:

### PR DESCRIPTION
**Pull request by thammarith**
Fixed #193

### CODE REVIEW

**Comment by thammarith** [comment]
Thank you for your comments. I'm new to contributing and your comments really help me.

- [x] Changed _Related_ to _Fixed_
- [x] Changed the name format to **Name name (@username)**
- [x] Changed the heading from _Translators_ to _Translations_ so the links in `message.json` files should work fine. (I'm not sure if I understood this correctly and did it right).

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- anirudh-modi ([@anirudh-modi](https://github.com/anirudh-modi))
+- ENT8R ([@ENT8R](https://github.com/ENT8R))
+- Flo Becker ([@beckerei](https://github.com/beckerei))
+- Josh Cooper ([@JoshCooperr](https://github.com/JoshCooperr))
+- Linus Brogan ([@linusaurusrex](https://github.com/linusaurusrex))
+- Metta Ong ([@ongspxm](https://github.com/ongspxm))
+- Michael Corwin ([@mcorwin22](https://github.com/mcorwin22))
+- OogieBoogieInJSON ([@OogieBoogieInJSON](https://github.com/OogieBoogieInJSON))
+- redagavin ([@redagavin](https://github.com/redagavin))
+- xlomonosvx ([@xlomonosvx](https://github.com/xlomonosvx))
```
Comment (rugk) [inline CONTRIBUTORS.md]: If there is only the username, you do not need to mention it twice. In this case, you can really directly link it via `[@user](…)`. (taht looks strange IMHO)

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Contributors
+
+- Alexander Knipping ([@obitech](https://github.com/obitech))
+- Amauri Bizerra ([@AmauriAires](https://github.com/AmauriAires))
+- anirudh-modi ([@anirudh-modi](https://github.com/anirudh-modi))
+- ENT8R ([@ENT8R](https://github.com/ENT8R))
+- Flo Becker ([@beckerei](https://github.com/beckerei))
+- Josh Cooper ([@JoshCooperr](https://github.com/JoshCooperr))
```
Comment (rugk) [inline CONTRIBUTORS.md]: BTW, @thammarith feel free to add yourself. :smile:

File: CONTRIBUTORS.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+- Linus Brogan ([@linusaurusrex](https://github.com/linusaurusrex))
+- Metta Ong ([@ongspxm](https://github.com/ongspxm))
+- Michael Corwin ([@mcorwin22](https://github.com/mcorwin22))
+- OogieBoogieInJSON ([@OogieBoogieInJSON](https://github.com/OogieBoogieInJSON))
+- redagavin ([@redagavin](https://github.com/redagavin))
+- xlomonosvx ([@xlomonosvx](https://github.com/xlomonosvx))
+
+## Translations
```
Comment (rugk) [inline CONTRIBUTORS.md]: > Changed the heading from Translators to Translations so the links

Ah no, sorry. That's not what I meant.

Actually, I just had a look and in all current translations there is not even such a link. (the one you possibly mean goes to the `CONTRIBUTING.md` file, that is unrelated to this one, here.) This is then only relevant for later translations.

So sorry, I guess "translators" is really the better term here, this file is about humans, after all… :wink: 
```suggestion
## Translators
```

**Comment by thammarith** [comment]
Done in e9e079e. :D

**Comment by thammarith** [comment]
Thanks! I was fun and I learnt a lot. Many thanks!

**Comment by rugk** [comment]
Good to hear. That change should be included in v1.6 BTW, scheduled to be released today or tomorrow. So last minute changes here… :wink:

**Pull request merged**